From 2b1224b8d878c0758a9eb8c0051f62fdc6f6987a Mon Sep 17 00:00:00 2001
From: Vignesh Raghavendra <vigneshr@ti.com>
Date: Wed, 21 Jun 2023 11:59:42 +0530
Subject: [PATCH 036/143] remoteproc: ti_k3_m4_remoteproc: Fix compiler warning
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix below warning by dropping unused var

drivers/remoteproc/ti_k3_m4_remoteproc.c: In function ‘k3_m4_rproc_start’:
drivers/remoteproc/ti_k3_m4_remoteproc.c:244:13: warning: variable ‘boot_addr’ set but not used [-Wunused-but-set-variable]
  244 |         u32 boot_addr;
      |             ^~~~~~~~~

Fixes: 4572e31ebde2 ("remoteproc: k4-m4: Add a remoteproc driver for M4F subsystem")
Signed-off-by: Vignesh Raghavendra <vigneshr@ti.com>
---
 drivers/remoteproc/ti_k3_m4_remoteproc.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/remoteproc/ti_k3_m4_remoteproc.c b/drivers/remoteproc/ti_k3_m4_remoteproc.c
index 3c6f2fca5af4..12970144a563 100644
--- a/drivers/remoteproc/ti_k3_m4_remoteproc.c
+++ b/drivers/remoteproc/ti_k3_m4_remoteproc.c
@@ -241,7 +241,6 @@ static int k3_m4_rproc_start(struct rproc *rproc)
 {
 	struct k3_rproc *kproc = rproc->priv;
 	struct device *dev = kproc->dev;
-	u32 boot_addr;
 	int ret;
 
 	if (kproc->ipc_only) {
@@ -254,7 +253,6 @@ static int k3_m4_rproc_start(struct rproc *rproc)
 	if (ret)
 		return ret;
 
-	boot_addr = rproc->bootaddr;
 	ret = k3_rproc_release(kproc);
 	if (ret)
 		goto put_mbox;
-- 
2.41.0

